LICENSE

```diff
@@ -1,28 +1,11 @@
-BSD 3-Clause License
+This software is dual-licensed to the public under the Mozilla Public License 2.0 (MPL 2.0) and the GNU General Public License version 2.0 (GPL 2.0).
 
-Copyright (c) 2023, Jinzhao
+You may use this software under the terms of either the MPL 2.0 or the GPL 2.0.
 
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
+For MPL 2.0, see:
+https://opensource.org/licenses/MPL-2.0
 
-1. Redistributions of source code must retain the above copyright notice, this
-   list of conditions and the following disclaimer.
+For GPL 2.0, see:
+https://opensource.org/licenses/GPL-2.0
```

Comment:
The license files should be distributed with the source code. Giving an URL is not enough.

The more appropriate approach is to 
1. Add a file `COPYRIGHT` to describe the copyright information.
2. Add `LICENSE-XXX` files for each license that the copyright file refers to.

See the Asterinas repository as an example of this practice.

Reply:
Revised. Please review it again